pyproject.toml

```diff
@@ -1,6 +1,5 @@
 [tool.isort]
 profile = "black"
-src_paths = ["src", "test"]
```

Comment:
We're not following this convention here: this repo is not a library but it still contains `coded_tools`.

Reply:
Now specifying `coded_tools` and `tests` there.